src/ophys_etl/transforms/create_slapp_inputs.py

```diff
@@ -0,0 +1,270 @@
+import argschema
+import json
+import numpy as np
+from typing import List, Any
+from pathlib import Path
+import marshmallow as mm
+from marshmallow import ValidationError
+
+from ophys_etl.schemas.fields import H5InputFile
+
+
+class ExperimentSchema(argschema.schemas.DefaultSchema):
+    experiment_id = argschema.fields.Int(
+        required=True,
+        description="experiment id")
+    local_ids = argschema.fields.List(
+        argschema.fields.Int,
+        required=False,
+        cli_as_single_argument=True,
+        description="which ROIs to select from this experiment")
+    binarized_rois_path = argschema.fields.InputFile(
+        required=False,
+        description="path to output from ophys_etl.convert_rois")
+    traces_h5_path = H5InputFile(
+        required=False,
+        description="path to traces file output from extract_traces")
+    movie_path = H5InputFile(
+        required=True,
+        description="path to mtion corrected movie")
+
+
+class SlappTransformInputSchema(argschema.ArgSchema):
+    global_id_offset = argschema.fields.Int(
+        required=True,
+        description="global ROI ids will start at this value")
```

Comment:
Probably helpful to add more info about why this ID is computed (what purpose the offset serves)

Reply:
done